File: app/src/main/java/com/zxn/mvvm/demo/MainAdapter.java
package com.zxn.mvvm.demo;


import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.viewholder.BaseViewHolder;

/**
 * Created by zxn on 2019/4/2.
 */
class MainAdapter extends BaseQuickAdapter<String, BaseViewHolder> {

    public MainAdapter() {
        super(R.layout.item_main);
    }

    @Override
    protected void convert(BaseViewHolder helper, String item) {

    }
}

File: app/src/main/java/com/zxn/mvvm/demo/MainAdapter.kt
package com.zxn.mvvm.demo

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.viewholder.BaseViewHolder

/**
 * Created by zxn on 2019/4/2.
 */
class MainAdapter : BaseQuickAdapter<String, BaseViewHolder>(R.layout.item_main) {

    override fun convert(helper: BaseViewHolder, item: String) {}
}